Adapter_Revit_UI/AdapterActions/Pull.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using Autodesk.Revit.UI;
+using BH.oM.Adapter;
+using BH.oM.Adapters.Revit;
+using BH.oM.Data.Requests;
+using System.Collections.Generic;
+
+namespace BH.UI.Revit.Adapter
+{
+    public partial class RevitUIAdapter
+    {
+
+        /***************************************************/
+        /****             Protected Methods             ****/
+        /***************************************************/
```

Comment:
Old stuff, can be removed

Reply:
Isn't that a common graphical pattern used across the whole organization since some 2-3 years? Whole repo is formatted like that - happy to stick to it as long as @FraserGreenroyd does not hit me with the compliance truncheon.

Reply:
If you want to be worried about compliance, I'd rather notice that you say "protected methods" above a `public` method like the Pull 😄 

Jokes apart, we used to include these headers in the Adapter Actions when they were all mashed together in the same file. If you have only one method in the `.cs` file, I think it's better to remove this header. If you have more than one method in the same file, then the header are needed. For Adapter actions, this latter should never be the case.

Reply:
The subject of the joke has been changed in the commit pushed before your comment 🙈 

Jokes apart, as you state above, you _think_ it is the right way to go. However, I do not agree, and I think most of BHoM people would not, taken into account that virtually every .cs file in the core repos (BHoM, BHoM_Engine) contains headers regardless the number of items in the body. I do not see the point in changing the long-standing, commonly used pattern as long as it is not agreed upon it and clearly communicated to do so.

Reply:
As discussed, I have changed the headers to make them more informative. Thanks!